mallard/client.py

```diff
@@ -151,3 +162,35 @@ def _clean(self, message) -> Optional[str]:
             embed.color = self.color
 
         await message.channel.send(embed=embed)
+
+    async def bot_info(self, channel):
+        logger.info("Displaying bot info.")
+
+        usage = (
+            "To use me, just do an `@` mention or use one of the "
+            "above, then tell me what you want to know about!"
+        )
+
+        embed = discord.Embed(type='rich', color=discord.Color.dark_purple())
+        embed.title = self.user.display_name
+        embed.description = '\n'.join((
+            f"**{__name__} version {__version__}**",
+            f"Created by members of the Programming Server.",
+            "",
+            "Enabled mentions for this server:",
+            '\n'.join(f'\\* `{mention}`' for mention in self.mentions),
+            "",
+            usage,
+        ))
+        embed.add_field(name="GitHub", value="https://github.com/strinking/mallard", inline=True)
+        embed.add_field(name="Discord Server", value="https://discord.gg/010z0Kw1A9ql5c1Qe", inline=True)
+        embed.set_thumbnail(url=self.user.avatar_url)
+        await channel.send(embed=embed)
+
+    async def megane_easter_egg(self, channel):
+        logger.info("Displaying qt megane grilz.")
+
+        embed = discord.Embed(type='rich', color=MAWARE_COLOR)
+        embed.set_footer(text=":)")
+        embed.set_image(url=MEGANE_URL)
+        await channel.send(embed=embed)
```

Comment:
why

Reply:
For fun! Just a little thing that people can enjoy if they find it.

Reply:
you took my nice mallard and made it an anime bot. teaches me a lesson for putting stuff into this organization

Reply:
make anime real again